example/package.json

```diff
@@ -9,7 +9,7 @@
   "dependencies": {
     "react": "16.3.1",
     "react-native": "0.55.4",
-    "react-native-share": "git+https://github.com/kkbhav/react-native-share.git"
```

Comment:
thanks

Reply:
if only it actually worked! relative path resulted in bundler confusion - pushed a change that at least references the official repo though